idaes/core/flowsheet_model.py

```diff
@@ -203,10 +203,11 @@ def visualize(self, model_name, **kwargs):
         """
         visualize(self, model_name, **kwargs)
 
-    def get_costing(self, module=costing, year=None):
+    def get_costing(self, module=costing, year=None, integer_n_units=False):
```

Comment:
We probably should add a method doc string here to explain what this method does and what the arguments are. Seeing as this is a public facing method, I can see someone trying to use help on this, so it would be good to provide some information. This would also end up in the autodocs for the flowsheet block class, meaning that people would see the costing methods there and maybe come looking for more details.